### DISCUSSION TITLE
Having issues with chaining the font size of the title in the plot_gate function.

### DISCUSSION THREAD

**Discussion by javisolivan** [whitews/FlowKit#170] [General]
I am working to generate plots for fcs files that I have generated. I am tying to export high resolution images but to do so I need to increase the font size. When doing so, all the fonts respond except the sample and gate path of my plot. I there a way to have the top middle text in the plot_gate function also increase in size? 

Here is the code I am using:

`
       output_notebook()
       png_files = []
       for index, den_sample_id in enumerate(wsp.get_sample_ids(group_name='All Samples')):
       p = wsp.plot_gate(den_sample_id, 'ChgA_Pos',x_min=0, x_max=1, y_min=0,y_max=1, bin_width=4)

    p.toolbar_location = None
    p.title = None
    p.plot_width = 1500  # Adjust this value as needed
    p.plot_height = 1500  # Adjust this value as needed
    p.xaxis.axis_label_text_font_size = '32pt'
    p.yaxis.axis_label_text_font_size = '32pt'
    p.xaxis.major_label_text_font_size = '32pt'
    p.yaxis.major_label_text_font_size = '32pt'

    sample_results = wsp.get_gating_results(den_sample_id)
    percentage_gated = sample_results.report[sample_results.report['gate_name'] == 'ChgA_Pos']['relative_percent'].iloc[0]
    # Annotate the Bokeh plot with the percentage
    percentage_label = Label(x=45, y=418, x_units='screen', y_units='screen',
                             text=f'{percentage_gated:.2f}%', render_mode='css')
    p.add_layout(percentage_label)
    # Export the individual plot to high-resolution PNG
    png_filename = f"plot_{index}.png"
    export_png(p, filename=png_filename, width=1500, height=1500)
    png_files.append(png_filename)
with PdfPages('combined_plots.pdf') as pdf:
    for png_file in png_files:
        img = plt.imread(png_file)
        plt.figure(figsize=(15, 15))
        plt.imshow(img)
        plt.axis('off')  # Turn off axis numbers and ticks
        pdf.savefig()  # saves the current figure into a pdf page
        plt.close()
`

![plot_0](https://github.com/whitews/FlowKit/assets/144291198/87c8421a-8508-421a-93be-4b90025381c9)

**Comment by whitews** [comment]
Hi,

It's kinda hidden but the titles are placed in a layout named `above`, and they can be accessed using `p.above`. The FCS file name is in a `Title` object in the 2nd element. To change the file name's font size:

p.above[1].text_font_size = '1.5em' 

Hope this helps!
-Scott
